tests/Application/webpack.config.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path');
```

Comment:
I would be for this improvement https://github.com/Sylius/Sylius-Standard/commit/e97cb7c578ac071ab6cb4438b252be1dbd85abe5 just to be consistent.